cli/cmd/profile.go

```diff
@@ -112,7 +95,6 @@ func newCmdProfile() *cobra.Command {
 			}
 			// performs an online profile generation and access-check to k8s cluster to extract
 			// clusterDomain from linkerd configuration
-			// profile generation based on tap data requires access to k8s cluster
 			if !options.ignoreCluster {
 				var err error
 				k8sAPI, err = k8s.NewAPI(kubeconfigPath, kubeContext, impersonate, impersonateGroup, 0)
```

Comment:
small nit, TIOLI: `k8sAPI` no longer needed for tap generation. we can make it a local variable right?

Reply:
Yep I can remove the `var k8sAPI *k8s.KubernetesAPI` above.